nvtabular/ops.py

```diff
@@ -1102,3 +1102,43 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
             new_gdf[col] = self.f(gdf[col], gdf)
         new_gdf.columns = [f"{col}_{self._id}" for col in new_gdf.columns]
         return new_gdf
+
+
+class Filter(TransformOperator):
+    """
+    Enables to call Methods to cudf.Series. Returns a subset of the original gdf.
```

Comment:
I think the docstring here could be made clearer: how about

```suggestion
    Filters rows from the dataset. This works by taking a callable that takes a dataframe, and returns 
    a dataframe with unwanted rows filtered out.
```
or something like that?